docs/resources/integration_aws_serverless.md

```diff
@@ -144,11 +144,14 @@ Optional:
 
 Required:
 
-- `ebs_scan_options` (Attributes) (see [below for nested schema](#nestedatt--scan_configuration--ec2_scan_options--ebs_scan_options))
+- `ebs_scan_options` (Attributes, Deprecated) (see [below for nested schema](#nestedatt--scan_configuration--ec2_scan_options--ebs_scan_options))
 
 Optional:
 
 - `ebs_volume_scan` (Boolean) Enable EBS volume scan.
+- `exclude_instance_ids_filter` (List of String) List of instance IDs to exclude.
+- `exclude_regions_filter` (List of String) List of regions to exclude.
+- `exclude_tags_filter` (Map of String) Excluded Tags filter.
```

Comment:
I wish we could have used a Set here to define a nested set of filters. Like:
```hcl
 
  filters {
    exclude_tags = [""]
    exclude_regions = [""]
    exclude...
  }
```
Maybe in a new major version we can do this.